services/ux-backend/handler/onboarding.go

```diff
@@ -0,0 +1,103 @@
+package handler
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+)
+
+func OnboardingTokenHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case "POST":
+
+		response, err := getOnboardingToken()
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		//nolint
+		w.Write(response)
+
+	default:
+		fmt.Fprintf(w, "Sorry, only POST methods are supported.")
+	}
+}
+
+func getOnboardingToken() ([]byte, error) {
+
+	expirationHour := os.Getenv("ExpirationHour")
+	if expirationHour == "" {
+		expirationHour = "48"
+	}
+
+	expireHour, err := strconv.Atoi(expirationHour)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Prefered way to read and check the value

```suggestion
	tokenLifetimeInHours, err := strconv.Atoi(os.Getenv("ONBOARDING_TOKEN_LIFETIME"))
	if err != nil {
		tokenLifetimeInHours = 48
	}
```